persistent-sqlite/persistent-sqlite.cabal

```
@@ -44,7 +44,7 @@ flag use-stat4
 
 library
     build-depends:   base                    >= 4.9         && < 5
-                   , persistent              >= 2.13.3      && < 3
+                   , persistent              >= 2.14.7      && < 3
```

```suggestion
                   , persistent              >= 2.15.0      && < 3
```

I'll make Hackage revisions on these packages to put upper bounds of `2.15` on them